python/TestHarness/testers/RunCommand.py

```diff
@@ -0,0 +1,30 @@
+import re, os, sys, time
+from Tester import Tester
+from RunParallel import RunParallel # For TIMEOUT value
+
+class RunCommand(Tester):
+
+  @staticmethod
+  def validParams():
+    params = Tester.validParams()
+    params.addRequiredParam('command',      "The command line to execute for this test.")
```

Comment:
There's no good way. That's why we run on machines that can be rebuilt if somebody doesn't something malicious.